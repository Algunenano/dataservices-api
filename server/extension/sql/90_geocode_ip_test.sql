-- Check that the public function is callable, even with no data
-- It should return NULL
SELECT cdb_geocoder_server.geocode_ip_point(session_user, txid_current(), '0.0.0.0');

-- Insert dummy data into ip_address_locations
INSERT INTO ip_address_locations VALUES ('0.0.0.0'::inet, (ST_SetSRID(ST_MakePoint('40.40', '3.71'), 4326)));

-- This should return the polygon inserted above
SELECT cdb_geocoder_server.geocode_ip_point(session_user, txid_current(), '0.0.0.0');
