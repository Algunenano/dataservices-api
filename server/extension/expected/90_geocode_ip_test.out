-- Check that the public function is callable, even with no data
-- It should return NULL
SELECT cdb_geocoder_server.geocode_ip_point(session_user, txid_current(), '0.0.0.0'::inet);
 geocode_ip_point 
------------------
 
(1 row)

-- Insert dummy data into ip_address_locations
INSERT INTO ip_address_locations VALUES ('0.0.0.0'::inet, (ST_SetSRID(ST_MakePoint('40.40', '3.71'), 4326)));
-- This should return the polygon inserted above
SELECT cdb_geocoder_server.geocode_ip_point(session_user, txid_current(), '0.0.0.0'::inet);
                  geocode_ip_point                  
----------------------------------------------------
 0101000020E61000003333333333334440AE47E17A14AE0D40
(1 row)

